### PR TITLE
Project post mike edits

### PR DESCRIPTION
**Pull request by mgelbart**
@ttimbers I shortened your draft significantly. Let's try to converge on something in your branch and then we'll send it to master.

### CODE REVIEW

File: _posts/2019-07-07-project-courses.md [hunk 1]
```diff
@@ -1,95 +1,64 @@
...
 
-From our experience with the University of British Columbia's Master of Data Science program we have come to the opinion that project-based courses are necessary for a well-rounded Data Science education. We believe this is because it gives the students to work in a more real-life situation where the "right" answer is less well defined, collaboration is required and many axes of Data Science are brought together. We also believe that students should have the opportunity to do more than 1 project in their Data Science program so that they get to repeatedly practice an entire project from beginning to end in a lower stakes environment before they are required to do so on the job. 
+--------
+[Tiffany Timbers](https://www.tiffanytimbers.com/) is Option Co-Director of the MDS Vancouver program and an Instructor in the UBC Department of Computer Science.
```
Comment (ttimbers) [inline _posts/2019-07-07-project-courses.md]:
```suggestion
[Tiffany Timbers](https://www.tiffanytimbers.com/) is Option Co-Director of the MDS Vancouver program and an Instructor in the UBC Department of Statistics.
```

Comment (ttimbers) [reply _posts/2019-07-07-project-courses.md]: I'm in the Stat dept @mgelbart 😉

**Review by ttimbers — COMMENTED** [review]
Looks good, once you address my one comment (which on GitHub.com you can do with a click of a button, I think...) I will accept the PR, do one more read through without the changes. And then you can re-review my original PR. 

Thanks for your heroic effort in editing this down!

File: _posts/2019-07-07-project-courses.md (hunk 1 above)
Comment (ttimbers) [reply _posts/2019-07-07-project-courses.md]: Also, given your heroic effort in editing, should you also be listed as an author on this post?

**Comment by mgelbart** [comment]
I pressed the button and it worked! You'll have to explain that to the rest of the team! 

I'm ready for you to merge this PR and then create one from your branch to master, which I will merge. Actually, before you create the PR to master, it would be helpful for you to rename the file so that it has today's date instead of 07-07.

File: _posts/2019-07-07-project-courses.md [hunk 2]
```diff
@@ -1,95 +1,64 @@
...
 
-From our experience with the University of British Columbia's Master of Data Science program we have come to the opinion that project-based courses are necessary for a well-rounded Data Science education. We believe this is because it gives the students to work in a more real-life situation where the "right" answer is less well defined, collaboration is required and many axes of Data Science are brought together. We also believe that students should have the opportunity to do more than 1 project in their Data Science program so that they get to repeatedly practice an entire project from beginning to end in a lower stakes environment before they are required to do so on the job. 
+--------
+[Tiffany Timbers](https://www.tiffanytimbers.com/) is Option Co-Director of the MDS Vancouver program and an Instructor in the UBC Department of Statistics.
```
Comment (ttimbers) [inline _posts/2019-07-07-project-courses.md:64]:
```suggestion
[Tiffany Timbers](https://www.tiffanytimbers.com/) is Option Co-Director of the MDS Vancouver program and an Instructor in the UBC Department of Statistics.

[Mike Gelbart](https://www.mikegelbart.com/) is Option Co-Director of the MDS Vancouver program and an Instructor in the UBC Department of Computer Science.
```

**Review by ttimbers — COMMENTED** [review]
Adding @mgelbart as author in the second byline

**Pull request merged**